packages/components/src/popover/index.tsx

```diff
@@ -476,7 +488,7 @@ const UnforwardedPopover = (
 			placement={ computedPlacement }
 			className={ classnames( 'components-popover', className, {
 				'is-expanded': isExpanded,
-				'is-alternate': isAlternate,
+				[ `is-${ computedVariant }` ]: !! computedVariant,
```

Comment:
if we change name to the `alternate` variant, should we still look into keeping  the `is-alternate` classname for that specific variant for legacy reasons?

Reply:
That's a good point. Technically we can change it according to the back compat doc:
https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/backward-compatibility.md#class-names-and-dom-updates

But it may not be worth it. It would be good to keep it in sync with the prop value to avoid unnecessary branching code. So that would be an argument for keeping 'alternate'.

Reply:
I'd be in favour of changing the `variant` name to something else (i.e. `high-contrast`), but then mapping that variant to the `is-alternate` classname for back compat.

Reply:
It's changed to 'toolbar' now as per the other conversation (https://github.com/WordPress/gutenberg/pull/45137#discussion_r1003126851), it still outputs the `is-alternate` classname.